src/.eslintrc

```diff
@@ -0,0 +1,20 @@
+{
+	"env": {
+		"browser": true
```

Comment:
This shouldn't be here for the same reason that we haven't specified `"browser": true` in `.jshintrc` for some time. We only assume the presence of the `window` global, the rest of the browser globals have to be taken from `window`. This is the main way in which we guarantee working correctly in configurations like Node+jsdom.